cmd/vault-staging-k8s-secret/main.go

```diff
@@ -42,45 +42,64 @@ func run(ctx context.Context, lgr *zap.Logger) error {
 
 	cfg := newConfig(lgr)
 
-	engines := []string{cfg.defaultEngine}
-	if cfg.override != "" {
-		engines = append(engines, cfg.override)
+	secretsToApply := []string{cfg.defaultSecretPath}
+	if cfg.overrideSecretPath != "" {
+		secretsToApply = append(secretsToApply, cfg.overrideSecretPath)
 	}
 
-	var verifiedEngines []string
-	for _, p := range engines {
-		lgr.Debug("verifying engine exists for path", zap.String("path", p))
-		if _, err := vault.ListEngines(ctx, p); err != nil {
-			verifiedEngines = append(verifiedEngines, p)
-		} else {
-			lgr.Error("cannot verify engine exists at path", zap.Error(err))
+	var vaultSecrets []string
+	var err error
+	lgr.Debug("getting secrets for engine", zap.String("engine", cfg.engine))
```

Comment:
is `engine` still the right terminology here?

Reply:
yes at this point in the code we're trying to validate the engine itself exists